### PR TITLE
Adjust yamlRestCompatTest to account for some tests that are passing

### PR DESCRIPTION
**Pull request by joegallo**
These tests are being ignored, but it looks to me like they actually pass -- probably because of other PRs that have already been merged.

### CODE REVIEW

File: rest-api-spec/build.gradle [hunk 1]
```diff
@@ -158,16 +143,13 @@ tasks.named("yamlRestCompatTest").configure {
...
     'get_source/86_source_missing_with_types/Missing document source with ignore',
-    'indices.clone/10_basic/Clone index via API',
     'indices.create/10_basic/Create index without soft deletes',
     // 5 below await retrofitting Removes typed URLs from mapping APIs #41676
     'indices.create/11_basic_with_types/Create index with mappings',
     'indices.create/20_mix_typeless_typeful/Create a typed index while there is a typeless template',
     'indices.create/20_mix_typeless_typeful/Create a typeless index while there is a typed template',
     'indices.create/20_mix_typeless_typeful/Implicitly create a typed index while there is a typeless template',
     'indices.create/20_mix_typeless_typeful/Implicitly create a typeless index while there is a typed template',
-    //
```
Comment (pgomulka) [inline rest-api-spec/build.gradle:169]: I use the empty // to indicate that the comment above affects the section of lines
probably not the best way to comment :D

**Review by pgomulka — APPROVED** [review]
LGTM
I actually thought that we could do this after mappings and search related apis 
I expect a lot of tests are affected because the "basic" typed create index, put mapping, index doc, search was removed.
But I am happy if we do this type of cleanups more often (I like seeing the number of blocked tests drop :D )

File: rest-api-spec/build.gradle (hunk 1 above)
Comment (joegallo) [reply rest-api-spec/build.gradle:169]: OOOHHHHH, okay, I'll fix that. Thanks!

Comment (joegallo) [reply rest-api-spec/build.gradle:169]: b93f636

**Pull request merged**